SafeguardSessions/test/Integration/TestGetData.query.pq

```diff
@@ -165,6 +175,49 @@ TestGetDataWithoutError = () =>
     in
         TestResultsContentWithoutStartTime(response, expectedData, expectedInfo);
 
+TestFirmwareCheckCanBeBypassed = () =>
+    let
+        infoResponse = FakeRawResponse([
+            body = [firmware_version = "unsupported_version"]
+        ]),
+        fakeResponses = [
+            #"https://dummy_url/api/audit/sessions?fields=%2A&sort=-start_time&snapshot=dummy_snapshot_id" = TestSafeguardSessions.Response.Last
+        ],
+        expectedData = #table(
+            type table [
+                session_id = text,
+                subtable.field1 = number,
+                subtable.field2 = text,
+                flatvalue1 = number,
+                flatvalue2 = logical
+            ],
+            {}
+        ),
+        expectedInfo = FetchInfo(

```

Comment:
I know in TestLogger.query.pq it is visible that FetchInfo contains the connector version, but I think at least at one occurrence in TestFetchSessions.query.pq or in TestGetData.query.pq we should also test that FetchInfo contains the right connector version.
So instead of instantiating FetchInfo we should construct a raw table where we can write out the connector version explicitly.

Reply:
I created a separate test file for FetchInfo.

Reply:
Great! Thanks!